README.rst

```diff
@@ -36,9 +36,9 @@ Features
 * Implementation of `C++20 std::format <https://fmt.dev/Text%20Formatting.html>`__.
 * Support for user-defined types.
 * High performance: faster than common standard library implementations of
-  `printf <http://en.cppreference.com/w/cpp/io/c/fprintf>`_ and
+  `printf <https://en.cppreference.com/w/cpp/io/c/fprintf>`_ and
   iostreams. See `Speed tests`_ and `Fast integer to string conversion in C++
-  <http://zverovich.net/2013/09/07/integer-to-string-conversion-in-cplusplus.html>`_.
+  <https://zverovich.net/2013/09/07/integer-to-string-conversion-in-cplusplus.html>`_.
```

Comment:
This one won't work over https.

Reply:
I saw `https://www.zverovich.net/2013/09/07/integer-to-string-conversion-in-cplusplus.html` works but with *mixed content*. Anyway I have changed it back to http.